sass-mixins/nitro-ui/forms/_multi-input-group.scss

```diff
@@ -43,10 +43,12 @@ $selectors: (
       width: auto;
     }
     .input-group-addon, &.input-group-addon {
-      width: auto;
-      display: block;
-      vertical-align: -webkit-baseline-middle;
-      padding-top: 9px;
+      &:not(.multi) {
+        width: auto;
+        display: block;
+        vertical-align: -webkit-baseline-middle;
+        padding-top: 9px;  
+      }
```

Comment:
@jasoncypret - This will allow an exception for `DatePicker` which does not need these styles when used with `multi-input-group`.